settings.gradle

```diff
@@ -61,6 +60,7 @@ include ':redwood-lazylayout-testing'
 include ':redwood-lazylayout-uiview'
 include ':redwood-lazylayout-view'
 include ':redwood-lazylayout-widget'
+include ':redwood-modifiers'
```

Comment:
I think this should have ended up as 'redwood-layout-modifiers' since it's modifiers that are specific to the 'layout' modules.

Reply:
✅